lms/static/sass/views/_login-register.scss

```diff
@@ -445,12 +446,13 @@
 
     select {
       background: transparent;
-      opacity: 0.85;
-      border: none;
-      outline: solid 1px $gray-l3;
+      border: 1px solid $gray;
```

Comment:
Can you add a link to the file where `$gray` is defined?

Reply:
https://github.com/edx/edx-platform/blob/master/lms/static/sass/partials/lms/theme/_variables-v1.scss

This is already declared variable so. I have used it instead of creating a new color variable.

Reply:
why did you removed the `opacity`?

Reply:
> why did you remove the `opacity`?

Opacity is removed because it lowers the color intensity of the border of `select`  so border color is not matched to other input fields.